### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by jtl5770** [lvjr/tabularray#465] [General]
I don't know where to ask this instead, but it seems that there is no progress anymore on the project for more or less a 3/4 of a year - is this project abandoned by the lead developer? Are others taking over?

I wonder because I need to decide if I should base my work on this library or not.
Sorry if this is not the best place to ask.

**Comment by lvjr** [comment]
See https://github.com/lvjr/tabularray/wiki/ChangeLog for the roadmap.

**Reply by lvjr** [reply]
See also https://github.com/lvjr/tabularray/milestone/9 .
